guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,29 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Assigning the Correct Organization and Location to {SmartProxyServer} in the {Project} web UI
+
+After installing the {SmartProxyServer} packages, you must enable the {SmartProxy} in the {Project} web UI by assigning the correct organization and location to it.
+
+.Procedure
+
+. Log into the {Project} web UI.
+. From the *Organization* list in the upper-left of the screen, select *Any Organization*.
+. From the *Location* list in the upper-left of the screen, select *Any Location*.
+. Navigate to *Hosts* > *All Hosts* and select the {SmartProxyServer}. 
+. From the *Select Actions* list, select *Assign Organization*. 
+. From the *Organization* list, select the organization where you want to assign this {SmartProxy}. 
+. Click *Fix Organization on Mismatch*. 
+. Click *Submit*. 
+. Select the {SmartProxyServer}. From the *Select Actions* list, select *Assign Location*.
+. From the *Location* list, select the location where you want to assign this {SmartProxy}.
+. Click *Fix Location on Mismatch*.
+. Click *Submit*.
+. Navigate to *Administer* > *Organizations* and click the organization to which you have assigned {SmartProxy}.
+. Click the *{SmartProxies}* tab and ensure that the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
+. Navigate to *Administer* > *Locations* and click the location to which you have assigned {SmartProxy}.
+. Click the *{SmartProxies}* tab and ensure that the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
+
+Select the organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations.
+
+* Navigate to *Hosts* > *All Hosts*.
+* Navigate to *Infrastructure* > *{SmartProxies}*.
```

Comment:
Sorry for all the nitpicking.
`Select the organization and location in the {Project} web UI, and verify if the {SmartProxyServer} is listed in the following locations.` - this is a procedural step, not a paragraph. 
 Bullets are not used for sequential steps.

I understand that the procedure is complete. Perhaps this might be better placed in a Conclusion section where they can optionally verify as this is not part of the procedure, and I can understand the difficulty you're having expressing this.

Reply:
Thanks, done.